github_connector_branch_options/models/github_repository_branch.py

```diff
@@ -0,0 +1,62 @@
+# Copyright Cetmix OU
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+
+from odoo import api, fields, models
+
+_odoo_version = [
+    "5.0",
+    "6.0",
+    "6.1",
+    "7.0",
+    "8.0",
+    "9.0",
+    "10.0",
+    "11.0",
+    "12.0",
+    "13.0",
+    "14.0",
+    "15.0",
+    "16.0",
+    "5",
+    "6",
+    "7",
+    "8",
+    "9",
+    "10",
+    "11",
+    "12",
+    "13",
+    "14",
+    "15",
+    "16",
```

Comment:
don't understand that values.

Reply:
if the odoo version is specified in the branch name in this format (not 12.0-my_branch-... but 12-my_branch-...)